back/spec/rails_helper.rb

```diff
@@ -94,3 +94,18 @@
 end
 
 ActiveJob::Base.queue_adapter = :test
+
```

Comment:
Hadn't meant to check this in yet - but this is a method I've added which I was going to use to help analyse where the performance blocks are within the permissions code

Reply:
Should we delete it now?